dir.props

```diff
@@ -15,6 +15,7 @@
 
   <!-- Common nuget properties -->
   <PropertyGroup>
+    <NuGetDir>$(PackagesDir)</NuGetDir>
```

Comment:
I don't think this is needed anymore. Or do you plan on doing something with it later on?